src/keycloak/index.js

```diff
@@ -9,14 +9,6 @@ let initOptions = {
     pkceMethod: 'S256'
 };
 
-keycloak.init(initOptions).then((auth) => {
-
-    if (!auth) {
-        throw "Auth error";
-    }
-
-}).catch(() => {
-    console.log("Authenticated Failed");
-});
+keycloak.init(initOptions);

```

Comment:
This change isn't required by the PR, but I think it's good. In the event of errors, exceptions are still thrown and logged, I just removed some pretty useless code.